### ISSUE TITLE
In news list the pagination path to page 1 is not correct.

### ISSUE THREAD

**Issue by Balindil** [georgringer/news#895]
In Typo3 v9 I have routing installed as described [here](Documentation/AdministratorManual/BestPractice/Routing/Index.rst).

Every link in pagination is configured as http://www.domain.com/news-list/page/2 and so on.

But if I want to go back to the first page the link is http://www.domain.com/news-list?tx_news_pi1%5Baction%5D=list&tx_news_pi1%5Bcontroller%5D=News&cHash=6215abffbe6d796c7f94b16dbe3168cf

**Comment by zelcode** [comment]
One of the solution (**check** `limitToPages`):

```yaml
routeEnhancers:
  PageTypeSuffix:
    type: PageType
    default: /
    suffix: /
    index: ''
    map:
      /: 0
      sitemap.xml: 1533906435
      rss.xml: 9818
  NewsList:
    type: Plugin
    limitToPages:
      - 1
    routePath: '/{@widget_0/currentPage}'
    namespace: tx_news_pi1
    aspects:
      '@widget_0/currentPage':
        type: StaticRangeMapper
        start: '1'
        end: '999'
  NewsDetail:
    type: Extbase
    limitToPages:
      - 7
    extension: News
    plugin: Pi1
    routes:
      -
        routePath: '/{news_title}'
        _controller: 'News::detail'
        _arguments:
          news_title: news
    defaultController: 'News::detail'
    aspects:
      news_title:
        type: PersistedAliasMapper
        tableName: tx_news_domain_model_news
        routeFieldName: path_segment
```
Also related and (maybe) should be closed:
https://github.com/georgringer/news/pull/809

**Comment by echavaillaz** [comment]
To solve this issue, you just have to override the paginate template and remove all the `addQueryStringMethod="GET"` occurences.

**Comment by esokoll2** [comment]
We face the same problem an add only two lines

```
      - routePath: '/'
        _controller: 'News::list'
```

Regarding the News Documentation you end up with something like this:

```
routeEnhancers:
  News:
    type: Extbase
    extension: News
    plugin: Pi1
    routes:
      - routePath: '/'
        _controller: 'News::list'
      - routePath: '/page-{page}'
        _controller: 'News::list'
        _arguments:
          page: '@widget_0/currentPage'
      - routePath: '/{news-title}'
        _controller: 'News::detail'
        _arguments:
          news-title: news
```
[....]
```
    defaultController: 'News::list'
    defaults:
      page: '0'
    aspects:
      news-title:
        type: PersistedAliasMapper
        tableName: tx_news_domain_model_news
        routeFieldName: path_segment
      page:
        type: StaticRangeMapper
        start: '1'
        end: '100'
```
[....]

**Comment by georgringer** [comment]
IMO this has been solved in the core with next release of 9.5.14
